files/build_templates/teamd.service.j2

```diff
@@ -2,6 +2,7 @@
 Description=TEAMD container
 Requires=updategraph.service
 After=updategraph.service
+After=syncd.service
```

Comment:
You can combine them into one line, i.e., `After=updategraph.service syncd.service`.

Reply:
Thanks. Changed.